lib/oxrdf/src/blank_node.rs

```diff
@@ -121,6 +125,29 @@ impl Default for BlankNode {
     }
 }
 
+#[cfg(feature = "serde")]
+impl Serialize for BlankNode {
+    #[inline]
+    fn serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        json!({ "value": self.as_str() }).serialize(serializer)
```

Comment:
I don't think you need serde_json for this. Either you can create some struct 
```rust
#[derive(Serialize)]
struct NamedNodeForSerialization {
   r#type: &str,
   value: &str
}
```
or use directly the `Serializer` API:
```rust
        let mut state = serializer.serialize_struct("NamedNode", 2)?;
        state.serialize_field("type", "bnode")?;
        state.serialize_field("value", &self.value)?;
        state.end()
```